src/module/rules/rule-element/item-alteration/alteration.ts

```diff
@@ -389,6 +392,24 @@ class ItemAlteration extends foundry.abstract.DataModel<RuleElementPF2e, ItemAlt
                 }
                 return;
             }
+            case "range-increment": {
+                const validator = ITEM_ALTERATION_VALIDATORS[this.property];
+                if (!validator.isValid(data)) return;
+                if (!(data.item instanceof WeaponPF2e) || !data.item.system.range) return;
+                const rangeIncrement = data.item.system.range as number;
```

Comment:
```suggestion
                const rangeIncrement: number = data.item.system.range;
```

I'm assuming the casting is because range is a set of numbers, not a specific number, and modifying it might have caused issues.